development-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+
+services:
+  web:
```

Comment:
We should bring up postgres here, and pass it an an env to the app

Reply:
On it

Reply:
@hkirat i have added above changes please have a look